### PR TITLE
Brains act more like dead humans on examine.

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

If a ghost is in it's brain, it will now indicate that the spark is very bright in the brain.

If the ghost is still connected, it will give the faint spark of life message.

If the brain is dnr / not a valid brain / ghost not connected, it will do the lifeless message.

## Why It's Good For The Game

Makes it more similar to the messages on dead people. Right now, people mainly look if a ghost is in a brain to tell if they want to revive someone, as they can only see if someone is in the brain or not.
Now people can see if someone is in the brain (good to revive), ghost connected (probably good to revived), or unrevivable, like the soulless body on examine

## Testing

Confirms that then brain worked with ghost in brain, ghost in round but not in brain, and ghost dnr / a brain without a ghost.

## Changelog
:cl:
tweak: Brains are more clear in if they are having a ghost available for revival or not.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/brain_item.dm [hunk 1]
```diff
@@ -49,9 +49,20 @@
 /obj/item/organ/internal/brain/examine(mob/user) // -- TLE
 	. = ..()
 	if(brainmob && brainmob.client)//if thar be a brain inside... the brain.
-		. += "You can feel the small spark of life still left in this one."
-	else
-		. += "This one seems particularly lifeless. Perhaps it will regain some of its luster later.."
+		. += "You can feel a bright spark of life in this one!"
```
Comment (S34NW) [inline code/modules/mob/living/carbon/brain/brain_item.dm:52]:
```suggestion
		. += "You can feel a bright spark of life in this one!"
		return
```

File: code/modules/mob/living/carbon/brain/brain_item.dm [hunk 2]
```diff
@@ -49,9 +49,20 @@
...
+	else if(brainmob && brainmob.mind)
```
Comment (S34NW) [inline code/modules/mob/living/carbon/brain/brain_item.dm:53]:
```suggestion
	if(brainmob && brainmob.mind)
```

**Review by S34NW — CHANGES_REQUESTED** [review]
guard clause my beloved

File: code/modules/mob/living/carbon/brain/brain_item.dm [hunk 3]
```diff
@@ -49,9 +49,21 @@
...
-	else
-		. += "This one seems particularly lifeless. Perhaps it will regain some of its luster later.."
+		. += "You can feel a bright spark of life in this one!"
+		return
+	if(brainmob && brainmob.mind)
+		var/foundghost = FALSE
+		for(var/mob/dead/observer/G in GLOB.player_list)
+			if(G.mind == brainmob.mind)
+				foundghost = TRUE
+				if(G.can_reenter_corpse == 0)
```
Comment (Sirryan2002) [inline code/modules/mob/living/carbon/brain/brain_item.dm:59]:
```suggestion
				if(G.can_reenter_corpse == FALSE)
```

File: code/modules/mob/living/carbon/brain/brain_item.dm [hunk 4]
```diff
@@ -49,9 +49,21 @@
 /obj/item/organ/internal/brain/examine(mob/user) // -- TLE
 	. = ..()
 	if(brainmob && brainmob.client)//if thar be a brain inside... the brain.
-		. += "You can feel the small spark of life still left in this one."
-	else
-		. += "This one seems particularly lifeless. Perhaps it will regain some of its luster later.."
+		. += "You can feel a bright spark of life in this one!"
+		return
+	if(brainmob && brainmob.mind)
```
Comment (Sirryan2002) [inline code/modules/mob/living/carbon/brain/brain_item.dm:54]:
```suggestion
	if(brainmob?.mind)
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**